### DISCUSSION TITLE
Pimcore X - some buttons are disabled for admin

### DISCUSSION THREAD

**Discussion by rigelsoft** [pimcore/pimcore#10918] [Q&A / Support] [answered]
Hi

 after updating to 10.2.3 we have found that some buttons [i.e. _Add_ in _Thumbnails settings_, _Add_ in _Custom reports]_ for admin user are disabled. 
![image](https://user-images.githubusercontent.com/6872311/143842291-33a74410-8dd8-48b2-ab22-a17afb50a79e.png)

Any help would be appreciated

**Comment by rigelsoft** [comment, marked as answer]
It seems it's a matter of (Upgrade Notes 10.2.0):

Important notice: writing configs to yaml is only supported if the kernel is in debug mode, because changes of the config need a rebuild of the container configuration.

**Reply by dvesh3** [reply]
Also, you can enable writing configs by switching to `settings-store` database storage. please see https://pimcore.com/docs/pimcore/current/Development_Documentation/Deployment/Configuration_Environments.html#page_Production-environment-with
